README.md

```
@@ -1,219 +1,236 @@
+| **Deploy Anywhere** | Runs on Cloudflare Workers, AWS, or locally with Bun/Deno — no lock-in, one command deploy. |
 
 ---
 
-## Creating a new Deco project
+## 🧩 Architecture
 
-A Deco project extends a standard Cloudflare Worker with our building blocks and
-defaults for MCP servers.\
-It runs a type-safe API out of the box and can also serve views — front-end apps
-deployed alongside the server.
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)